src/Microsoft.DocAsCode.Build.Engine/DocumentBuildParameters.cs

```diff
@@ -10,20 +10,46 @@ namespace Microsoft.DocAsCode.Build.Engine
 
     public sealed class DocumentBuildParameters : MarshalByRefObject
     {
+        [IncrementalCheckAttribute]
         public FileCollection Files { get; set; }
```

Comment:
nice

---
In reply to: [75577824](https://github.com/dotnet/docfx/pull/625#discussion_r75577824) [](ancestors = 75577824)